applications/resturant/endpoints.py

```
@@ -436,6 +458,24 @@
     serializer_class = MenuSerializer
     permission_classes = [IsAuthenticated]
 
+class MenuCreateView(CachedResponseMixin, generics.CreateAPIView):
```

Ruff (0.11.9)

Redefinition of unused `MenuCreateView` from line 454

(F811)